universal-login-react/src/ui/Transfer/Amount/TransferDropdown.tsx

```diff
@@ -34,7 +34,7 @@ export const TransferDropdown = ({sdk, tokenDetailsWithBalance, tokenDetails, se
       className={className}
       tokenDetails={tokenDetails}
       icon={getIconForToken(tokenDetails.symbol)}
-      balance={getBalanceOf(tokenDetails.symbol, tokenDetailsWithBalance)}
+      balance={ValuePresenter.presentRoundedValue((getBalanceOf(tokenDetails.symbol, tokenDetailsWithBalance)!), ValueRounder.floor)}
```

Comment:
Move usingPresenter into getBalanceOf() => It will change all balances properlywithout using it in each case